cmd/metrics-server/app/options/kubelet_client.go

```diff
@@ -40,26 +40,26 @@ type KubeletClientOptions struct {
 func (o *KubeletClientOptions) Validate() []error {
 	errors := []error{}
 	if (o.KubeletCAFile != "") && o.InsecureKubeletTLS {
-		errors = append(errors, fmt.Errorf("Cannot use both --kubelet-certificate-authority and --kubelet-insecure-tls"))
+		errors = append(errors, fmt.Errorf("cannot use both --kubelet-certificate-authority and --kubelet-insecure-tls"))
```

Comment:
Is there a standard for log output? I'm a little confused.
Log structured migration requires `Start from a capital letter.`

Reply:
Golang specifies that errors should start from small letter.